app/controllers/schools/setup_school_cohort_controller.rb

```diff
@@ -168,7 +195,19 @@ def set_cohort_induction_programme!(programme_choice, opt_out_of_updates: false)
       Induction::SetCohortInductionProgramme.call(school_cohort:,
                                                   programme_choice:,
                                                   opt_out_of_updates:,
-                                                  delivery_partner_to_be_confirmed: delivery_partner_to_be_confirmed?)
+                                                  delivery_partner_to_be_confirmed: delivery_partner_to_be_confirmed?,
+                                                  appropriate_body_appointed: appropriate_body_appointed?,
+                                                  appropriate_body: @setup_school_cohort_form.appropriate_body)
+    end
+
+    def appropriate_body_appointed?
+      @setup_school_cohort_form.appropriate_body_appointed == "yes"
+    end
+
```

Comment:
Could we move this method onto the form?

Reply:
👍